### ISSUE TITLE
grpc: received message larger than max

### ISSUE THREAD

**Issue by derkoe** [gavinbunney/terraform-provider-kubectl#59]
Just ran into this error with the cert-manager (26k lines of yaml).

Seems to be fixed in Terraform (https://github.com/hashicorp/terraform/issues/20879) but the version has to be updated by each provider.

Can be reproduced with the current [cert-manager.yaml](https://github.com/jetstack/cert-manager/releases/download/v1.0.3/cert-manager.yaml) and:

```hcl
data "kubectl_file_documents" "cert_manager" {
  content = file("${path.module}/cert-manager.yaml")
}

resource "kubectl_manifest" "cert_manager" {
  count     = length(data.kubectl_file_documents.cert_manager.documents)
  yaml_body = element(data.kubectl_file_documents.cert_manager.documents, count.index)
}
```

Run `terraform apply` and then a `terraform plan` without changing anything.

**Comment by derkoe** [comment]
As a workaround for cert-manager you can use the Helm Chart for installing

**Comment by pit** [comment]
Could it be somehow connected to https://github.com/hashicorp/terraform/pull/20906 ?

**Comment by gavinbunney** [comment]
Hi @derkoe, I've pushed a fix for this in v0.9.3 (latest) of the provider. Confirmed that cert-manager now installs happily without running into the size issues (just hit this today upgrading cert-manager from v0.15.0 to v1.1.0).

Also sent an upgrade PR to https://github.com/hashicorp/terraform-plugin-sdk/pull/650 to fix this for all plugins.
